Library/Homebrew/upgrade.rb

```diff
@@ -193,6 +193,7 @@ def install_formula(formula_installer, upgrade:)
 
       if upgrade
         print_upgrade_message(formula, formula_installer.options)
+        Install.puts_no_install_upgrade_disable_message_if_not_already!
```

Comment:
This doesn't seem to exist?

Reply:
Thanks, good catch, left from refactoring out some of these.